eiger.cscs.ch/multicore-mr0-debug/computer-setup.yaml

```diff
@@ -7,11 +7,13 @@ work_dir: "/scratch/e1000/{username}/aiida/"
 shebang: "#!/bin/bash"
 mpirun_command: "srun -n {tot_num_mpiprocs}"
 mpiprocs_per_machine: 128
+num_cores_per_mpiproc: 1
```

Comment:
Just out of curiosity: why is this addition necessary?

Reply:
So would you suggest removing this line entirely and adding explicitly `export OMP_NUM_THREADS=1` into prepend text?

Reply:
I'm not sure how else to fix this. I double checked, and you indeed need to specify `--cpus-per-task` in order to have the `SLURM_CPUS_PER_TASK` env variable be set:

https://slurm.schedmd.com/sbatch.html#OPT_SLURM_CPUS_PER_TASK

And the computer setup command only uses `mpiprocs_per_machine`. So either we set `export OMP_NUM_THREADS=1` or `metadata.options.resources.num_cores_per_mpiproc` must be set.

Reply:
I tried doing the math in the submission file:

```
export CPUS_PER_TASK=$((SLURM_CPUS_ON_NODE / SLURM_NTASKS_PER_NODE))
```

But it seems that `SLURM_CPUS_ON_NODE` returns the number of _threads_, since `CPUS_PER_TASK` turns out to be 2 in this case. We could still divide by two I guess, but maybe setting `export OMP_NUM_THREADS=1` to the prepend text of the code is more sensible.